pkg/util/mount/mount_linux.go

```diff
@@ -240,13 +248,31 @@ func readProcMountsFrom(file io.Reader, out *[]MountPoint) (uint32, error) {
 func (mounter *SafeFormatAndMount) formatAndMount(source string, target string, fstype string, options []string) error {
 	options = append(options, "defaults")
 
+	// Run fsck on the disk to fix repairable issues
+	args := []string{"-a", source}
+	cmd := mounter.Runner.Command("fsck", args...)
```

Comment:
mount does not do a search: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_linux.go#L99

nsenter_mount does. I misread when I made my last comment. The issue there must have been an unset $PATH since the path it is searching is fairly standard.

Anyhow if a path search is necessary I think it should be done in `util/exec` instead of duplicating the code from nsenter_mounter.go here. WDYT ?